### PR TITLE
Add editor preview images displaying

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add object editor preview images displaying (like 3DEN) when an item is hovered in object trees
- Can be disabled through the setting

<details>
<summary>Image</summary>

editor_previews

</details>

### CODE REVIEW

File: addons/editor_previews/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ZEN">
+    <Package name="Editor_Previews">
+        <Key ID="STR_ZEN_Editor_Previews_Enabled">
+            <English>Editor Previews</English>
+        </Key>
+        <Key ID="STR_ZEN_Editor_Previews_Enabled_Description">
+            <English>Enables the displaying of object preview images when an item is hovered in the create trees.</English>
```
Comment (CreepPork) [inline addons/editor_previews/stringtable.xml]: Create tree. There’s only one (if you don’t count the sub ones)

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Special thanks to the [ACE3 Team](http://ace3mod.com/team.html) for their open s
 - Overhauled markers tree with markers sorted into categories.
 - Ability to create and edit area markers through Zeus.
 - Rewritten, faster remote controlling of units.
+- Object editor preview images displaying in Zeus.
```
Comment (neilzar) [inline README.md]:
```suggestion
- 3den editor object preview images in Zeus.
```

File: addons/editor_previews/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ZEN">
+    <Package name="Editor_Previews">
+        <Key ID="STR_ZEN_Editor_Previews_Enabled">
+            <English>Editor Previews</English>
```
Comment (neilzar) [inline addons/editor_previews/stringtable.xml]:
```suggestion
            <English>Object Preview Images</English>
```

File: addons/editor_previews/stringtable.xml (hunk 1 above)
Comment (neilzar) [inline addons/editor_previews/stringtable.xml]:
```suggestion
            <English>Enables object preview images when hovering over an item in the create trees.</English>
```

File: docs/home.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ Special thanks to the [ACE3 Team](http://ace3mod.com/team.html) for their open s
 - Overhauled markers tree with markers sorted into categories.
 - Ability to create and edit area markers through Zeus.
 - Rewritten, faster remote controlling of units.
+- Object editor preview images displaying in Zeus.
```
Comment (neilzar) [inline docs/home.md]:
```suggestion
- 3den editor object preview images in Zeus.
```

**Pull request merged**